### PR TITLE
ProcessManager: Handle Monitor#slow in subprocesses

### PR DESCRIPTION
**Pull request by mia-pi-git**
Fixes ``A simulator process crashed: TypeError: Monitor.slow is not a function``

### CODE REVIEW

File: lib/process-manager.ts [hunk 1]
```diff
@@ -124,6 +124,11 @@ export class QueryProcessWrapper<T, U> implements ProcessWrapper {
 				throw error;
 			}
 
+			if (message.startsWith(`SLOW\n`)) {
+				Monitor.slow(message.slice(5));
```
Comment (Zarel) [inline lib/process-manager.ts:128]: `Monitor` is not a Node.js global and can't be called from inside PM.

What we might want is to set up a listener inside PM for custom debug messages, which we could customize from the server.

**Comment by DaWoblefet** [comment]
@Zarel apparently side servers are reporting lots of crashes that this PR would fix; it hasn't been reported publicly but apparently lots of people are encountering it.

**Comment by mia-pi-git** [comment]
> @Zarel apparently side servers are reporting lots of crashes that this PR would fix; it hasn't been reported publicly but apparently lots of side server owners are encountering it.

(this also _has_ been an issue on main, search `TypeError: Monitor.slow is not a function` in crashlog emails)

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**